desktop-widgets/preferences/preferences_reset.cpp

```diff
@@ -0,0 +1,45 @@
+// SPDX-License-Identifier: GPL-2.0
+#include "preferences_reset.h"
+#include "ui_preferences_reset.h"
+#include "core/dive.h"
+#include "preferencesdialog.h"
+//#include "core/settings/qPrefGeneral.h"
+//#include "core/settings/qPrefDisplay.h"
+//#include "core/settings/qPrefCloudStorage.h"
+//#include "core/settings/qPrefDiveComputer.h"
```

Comment:
and more commented out include files.

Reply:
Thanks for the comments. If you would be prepared to rebase this, **I would appreciate it very much**. At this stage the combined work is so complex I would struggle to undo a lot of it and maintain coherence without ending with one single huge commit. Almost all of the commented lines needed removal. My approach was to comment out code but not remove it before the newly-written alternative stuff is fully functional: this way I cannot easily lose original code.

Reply:
as I said somewhere... I understand doing this during development... just don't commit those commented lines.
`git add -p <filename>` is your friend.
I'll refactor this in the next couple of hours and push it into a new branch and put a pointer into this PR